xrmocap/model/mvp/projattn.py

```diff
@@ -50,7 +49,14 @@ def __init__(self,
                 per feature level. Defaults to 4.
             projattn_pos_embed_mode (str, optional):
                 The positional embedding mode of projective
-                attention. Defaults to 'use_rayconv'.
+                attention. The mode includes 'use_rayconv',
+                'use_2d_coordconv' and 'ablation_not_use_rayconv'.
+                'use_raycon' uses camera ray dirction asd the positional
```

Comment:
asd -> as?

Reply:
Fixed in [e47f350](https://github.com/openxrlab/xrmocap/pull/21/commits/e47f35020fa26325b63a01427876cffed8cae452).